CHANGELOG.md

```diff
@@ -6,9 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [23.1.0]
+## [24.0.0]
 ### Added
-- add checkAndUpdateSingleCollectibleOwnershipStatus method ([#669](https://github.com/MetaMask/controllers/pull/669))
+- **Breaking** Adds checkAndUpdateSingleCollectibleOwnershipStatus method and changes method previously named checkAndUpdateCollectibleOwnershipStatus to checkAndUpdateAllCollectibleOwnershipStatus  ([#669](https://github.com/MetaMask/controllers/pull/669))
+  - Previously incorrectly released as minor version bump 23.1.0.
+  - Consumers who consumed checkAndUpdateCollectibleOwnershipStatus must update it's name to checkAndUpdateAllCollectibleOwnershipStatus.
```

Comment:
This seems like two distinct changes: a new method, and a rename. I've separated them into two change entries, with the breaking change under `Changed`.

I also changed the tense to keep it as imperative, and formatted the `**BREAKING**` part properly as it has been used in this changelog previously.

```suggestion
- Add `checkAndUpdateSingleCollectibleOwnershipStatus` method ([#669](https://github.com/MetaMask/controllers/pull/669))

### Changed
- - **BREAKING**:  Rename `checkAndUpdateCollectibleOwnershipStatus` to `checkAndUpdateAllCollectibleOwnershipStatus`  ([#669](https://github.com/MetaMask/controllers/pull/669))
  - Previously incorrectly released as minor version bump 23.1.0.
  - Consumers who used `checkAndUpdateCollectibleOwnershipStatus` must update it's name to `checkAndUpdateAllCollectibleOwnershipStatus`.
```

Reply:
I also changed the method names to be within backticks. I find that it makes the sentences easier to read when the code references are in backticks, so that my brain doesn't try to parse them as words then backtrack once I realize it's a code reference.

Reply:
Thanks for the cleanup!